src/components/BaseQuestionnaireResponseForm/widgets/ReferenceRadioButton/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Form, Radio, Space } from 'antd';
+import { Resource } from 'fhir/r4b';
+import _ from 'lodash';
+
+import { getDisplay } from 'src/utils';
+
+import { useReferenceRadioButton } from './hooks';
+import { AnswerReferenceProps } from '../reference';
+
+function ReferenceRadioButtonUnsafe<R extends Resource = any, IR extends Resource = any>(
+    props: AnswerReferenceProps<R, IR>,
+) {
+    const { questionItem } = props;
+    const { choiceOrientation = 'vertical' } = questionItem;
+
+    const { loadedOptions, fieldController } = useReferenceRadioButton(props);
+
+    const { formItem, value, onChange, disabled } = fieldController;
+
+    return loadedOptions ? (
```

Comment:
@qscgyjqscgyj Please use RenderRemoteData to represent loading state for the widget.